build/includes/sdk.mk

```diff
@@ -113,12 +113,14 @@ ensure-build-sdk-image:
 # SDK client test against it. Useful for test development
 SECONDS ?= 30
 run-sdk-conformance-local:
+run-sdk-conformance-local: ensure-agones-sdk-image
```

Comment:
Is line 115 (in the new diff) now unnecessary? Or should this new part be on 115 instead of 116?

Reply:
removed redundant line